frontend/smart-kickers-game/src/tests/App.spec.js

```diff
@@ -2,6 +2,8 @@ import { render } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import WS from 'jest-websocket-mock';
 import { getElementWhichContain } from './helpers';
+jest.mock('../apis/Game');
+import * as GameAPI from '../apis/Game';
```

Comment:
In my opinion for cleaner code this `jest.mock` should be under imports

Reply:
Is it even possible? As far as I remember Jest needs to setup those mocks before the actual imports.